src/pages/News.jsx

```diff
@@ -1,91 +1,134 @@
-import { useState } from "react"
-
+import { useState, useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+import dayjs from "dayjs"
+import "dayjs/locale/ko"
 import styled from "@emotion/styled"
-
 import Stack from "@mui/material/Stack"
 import Typography from "@mui/material/Typography"
-import CampaignOutlinedIcon from "@mui/icons-material/CampaignOutlined"
+import Chip from "@mui/material/Chip"
 import Accordion from "@mui/material/Accordion"
 import AccordionSummary from "@mui/material/AccordionSummary"
 import AccordionDetails from "@mui/material/AccordionDetails"
 import IconButton from "@mui/material/IconButton"
-import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos"
-import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos"
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft"
+import ArrowRightIcon from "@mui/icons-material/ArrowRight"
 import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown"
 
 import Layout from "../components/Layout/Layout"
-import Title from "../components/shared/Title"
-import { NewsData } from "../data/NewsData"
+import { PlusButton } from "../components/shared/PlusButton"
+import { getMcyNewsApi } from "../api/newsApi"
+import { collection, deleteDoc, getDocs } from "firebase/firestore"
+import { db } from "../firebase" // firebase 설정 파일을 임포트
 
 const News = () => {
-  const currentDate = new Date()
-  const currentYear = currentDate.getFullYear()
-  const currentMonth = currentDate.getMonth() + 1
+  // 첫 렌더링시 파이어베이스에서 데이터 가져오기
+  const [newsData, setNewsData] = useState([])
+  useEffect(() => {
+    // 파이어베이스에서 News 콜레션에 있는 데이터를 가져옴.
+    const fetchData = async () => {
+      try {
+        const data = await getMcyNewsApi()
+        setNewsData(data)
+      } catch (error) {
+        console.error("Error fetching data: ", error)
+      }
+    }
 
-  const [selectedYear, setSelectedYear] = useState(currentYear)
-  const [selectedMonth, setSelectedMonth] = useState(currentMonth)
+    fetchData()
+  }, [])
+  // 날짜 이동
+  const currentDate = dayjs()
+  const [selectedDate, setSelectedDate] = useState(currentDate)
+  const selectedYear = selectedDate.year()
+  const selectedMonth = selectedDate.month() + 1 // Day.js의 month()는 0부터 시작하므로 1을 더합니다.
 
   const handlePreviousMonth = () => {
-    let newYear = selectedYear
-    let newMonth = selectedMonth - 1
-    if (newMonth === 0) {
-      newMonth = 12
-      newYear--
-    }
-    setSelectedYear(newYear)
-    setSelectedMonth(newMonth)
+    const newDate = selectedDate.subtract(1, "month")
+    setSelectedDate(newDate)
   }
 
   const handleNextMonth = () => {
-    let newYear = selectedYear
-    let newMonth = selectedMonth + 1
-    if (newMonth === 13) {
-      newMonth = 1
-      newYear++
+    const newDate = selectedDate.add(1, "month")
+    setSelectedDate(newDate)
+  }
+
+  // 파이어베이스 및 페이지 소식 데이터 삭제
+  const handleDelete = async (itemId, itemYear, itemMonth, itemDay) => {
```

Comment:
```suggestion
  const handleDelete = async (targetId, targetYear, targetMonth, targetDay) => {
```
네이밍은 target이 나을 것 같고, id값을 가지고 있으면 id는 유니크한 값일 텐데 id로만 찾아서 삭제해도될거같음. 굳이 년월일을 가지고 다닐필요가 없음